server/channels/app/config.go

```diff
@@ -214,11 +214,24 @@ func (a *App) GetConfigFile(name string) ([]byte, error) {
 // GetSanitizedConfig gets the configuration for a system admin without any secrets.
 func (a *App) GetSanitizedConfig() *model.Config {
 	cfg := a.Config().Clone()
-	cfg.Sanitize()
+
+	a.SanitizedConfig(cfg)
 
 	return cfg
 }
 
+// SanitizedConfig sanitizes a given configuration for a system admin without any secrets.
+func (a *App) SanitizedConfig(cfg *model.Config) {
+	manifests, err := a.GetPluginManifests()
+	if err != nil {
+		// GetPluginManifests might error, e.g. when plugins are disabled.
+		// Sanitze all plugin settings in this case.
+		cfg.Sanitize(nil)
```

Comment:
I think we should return here. Otherwise, there's no point in checking the error, which is fine as well if we can guarantee `GetPluginManifests` will always return `nil`.